pets/cassandra/on-start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Parse out hostname, formatted as: petset_name-index
+IFS='-' read -ra ADDR <<< "$(hostname)"
```

Comment:
He means extracting the index from the hostname. 
Actually I'm using it as a cheap way to find my own DNS name since I don't know the cluster domain in here (the peer-finder gives me a sorted list of peer DNS names, I'm assuming I'm nth in the list, where "n" is extracted from hostname). strings.HasPrefix(hostname, DNSname is a better way and that's what I'm doing in my zookeeper pr (in bash of course).